app/views/citizens/income_summary/_add_other_income.html.erb

```diff
@@ -0,0 +1,9 @@
+<li>
+  <h2 class="app-task-list__section">
+    <span class="app-task-list__section-number">
+      <img src="/assets/plus_icon.svg" />
+    </span>
+    <%= link_to t('.add_other_income'), citizens_identify_types_of_income_path, 'aria-describedby' => 'eligibility-completed' %>
```

Comment:
Do we need a `aria-describedby` attribute?
Is `eligibility-completed` relevant here?

Reply:
Well, I was just taking it from the prototype.  I think we're going to have to put in a load of aria-described-by once we do an accessibiltiy review, but I'll take it out for now.